### DISCUSSION TITLE
##求救！！！！！！！！

### DISCUSSION THREAD

**Discussion by EllenJoe14** [share121/inter-knot#354] [其他]
#芭莱大厦在进大门后前进100米后上一号电梯 我在二楼走廊以壁画《想笑的莫娜丽莎》开始从左数第八间房的柜子里！外面全是以骸我的萝卜也不见了。。。。。求各位绳匠呆地们救一下。。我还年轻我不想死。

**Comment by IX-11** [comment]
这个委托我接了！
